### PR TITLE
File organizer

### PR DESCRIPTION
**Pull request by adedayoprcs**
I created a File Organizer.

### CODE REVIEW

File: File_Organizer/file-organizer.py
Comment (iamwatchdogs) [inline File_Organizer/file-organizer.py:1]: Please remove the additional empty new lines.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please @adedayoprcs make the requested changes.

File: File_Organizer/file-organizer.py
Comment (adedayoprcs) [reply File_Organizer/file-organizer.py:1]: Thank you for the correction, I've made the changes

**Review by iamwatchdogs — APPROVED** [review]
Requested changes have been made. Reviewed and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing @adedayoprcs, make sure to check your contributions on [GitHub Pages](https://grow-with-open-source.github.io/Python-Projects/).

**Pull request merged**